### ISSUE TITLE
system variables

### ISSUE THREAD

**Issue by chuck-john** [Iowa-Elixir/reunions#15]
How are you guys setting system variables?  I know about the `config/prod.secret.exs`, but I haven't seen documentation around a development environment equivalent.  Currently, I've got a `.env` with `DB_USERNAME` and `DB_PASSWORD` loaded through the `Procfile`, but I'm wondering if these should be set outside the app.  There doesn't seem to be much of a consensus online, and it seems wasteful to bring in `dotenv` or a similar dependency to manage it.  Anyway, I just wanted to get a poll started.  Feel free to drop a comment with your workflow.

Procfile
```
vars: source .env
web: MIX_ENV=prod mix phoenix.server
```

.env
```
export DB_USERNAME="root"
export DB_PASSWORD="password"
```

**Comment by joseluistorres** [comment]
Right now just using my local with ENV vars setting them manually 🙄 I think it will be nice something like https://github.com/trestrantham/ex_figaro but I think is just like `dotenv` no?
There are some other options here:
https://github.com/h4cc/awesome-elixir#configuration

**Comment by codebycliff** [comment]
Yeah, I'd like to see something as well. Something similar to `dotenv` would be better than managing the environment variables on our own.

**Comment by chuck-john** [comment]
Thank you, gentlemen!  I'll investigate a few packaged solutions this week.
